keps/sig-storage/1698-generic-ephemeral-volumes/README.md

```diff
@@ -370,12 +370,11 @@ automatically enable late binding for PVCs which are owned by a pod.
   well as negative case (feature disabled or feature used incorrectly).
 - The ephemeral volume test [was
   extended](https://github.com/kubernetes/kubernetes/commit/2468a24b7a732fa492027a159ee15b6d31bf0577#diff-20517805986874f69e3254bed93d59996d5a6fd571a70ab8120736ded5aafa24)
-  to also test generic ephemeral volumes in combination with all
-  drivers that support dynamic volume provisioning. It currently runs
-  as part of the
-  [alpha-canary-on-master](https://k8s-testgrid.appspot.com/sig-storage-csi-ci#alpha-canary-on-master)
-  Prow job. After the promotion to beta it will also be part of all
-  `-on-1.21` jobs.
+  to also test generic ephemeral volumes in combination with all drivers that
+  support dynamic volume provisioning. It runs as part of jobs for Kubernetes
+  >= 1.21, like
+  [1.21-test-on-1.21](https://testgrid.k8s.io/sig-storage-csi-ci#1.21-test-on-1.21)
+  ("Testpattern: Generic Ephemeral-volume").
 
 ### Graduation Criteria
```

Comment:
I just thought of a few more test cases we should cover:
- Since this feature impacts intree volume types, it would be good to make sure we test against in-tree drivers with CSI migration on. We can work with @Jiawei0227 to enable the testing.
- Do we have an e2e test for the volume limits scheduler logic that fixed last release?

Reply:
Both are valid points and will need further work. But they don't block merging of this PR, right?

We can target both as part of the implementation work for GA in 1.23. I'm adding them to the project board.

Reply:
yup sounds good to me. @jsafrane also had some ideas on enabling more tests.